src/pages/reader.vue

```diff
@@ -127,15 +127,15 @@ export default {
       await this.fetchISCNMetadata();
       await this.restoreSession();
       // TODO: use loginAddress
-      if (this.getAddress) {
+      if (this.getSessionWallet) {
```

Comment:
There is a watch property in L95, should we change it too?

https://github.com/likecoin/liker-land/blob/21ac94b5009f76784bb5dacff5c605cac8fc3021/src/pages/reader.vue#L93-L98

Reply:
[🎨 Watch sessionWallet to update owner info in /reader](https://github.com/likecoin/liker-land/pull/1823/commits/25189dbec682989b1f272679ff16e5a115eaef8d)